ValidationLibrary.Rules.Tests/HasReadmeRuleTests.cs

```diff
@@ -51,10 +51,12 @@ public void RuleName_IsDefined()
             Assert.NotNull(_rule.RuleName);
         }
 
-        [Test]
-        public async Task IsValid_ReturnsOkWhenReadmeExists()
+        static string[] ValidReadMeNames = { "ReAdMe.md", "rEaDmE.txt", "README", "readme", "ReAdMe.doc" };
+
+        [TestCaseSource("ValidReadMeNames")]
+        public async Task IsValid_ReturnsOkWhenReadmeExists(string readMeName)
```

Comment:
Use following show our analysis doesn't complain about unused values etc.

```
[Test]
public async Task IsValid_ReturnsOkWhenReadmeExists([Values("ReAdMe.md", "rEaDmE.txt", "README", "readme", "ReAdMe.doc")]string readMeName)
```

Reply:
What analysis is this? I don't get any warnings when building or testing

Reply:
Intellisense complains that these are not in use because it doesn't recognize that [TestCaseSource("ValidReadMeNames")] as usage.

Reply:
Oh I see you must be using Visual Studio or something. Mine doesn't complain but the machines are always right ;)

Reply:
Hmm, I'm using Visual Studio Code, what are you using?